### PR TITLE
feat: add remaning question components UI

### PR DESCRIPTION
**Pull request by Hoanqnqu**

### CODE REVIEW

File: tools/graphql.codegen.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import type { CodegenConfig } from '@graphql-codegen/cli';
 
 const config: CodegenConfig = {
 	overwrite: true,
-	schema: 'https://enode.peakee.co/api/graphql',
+	schema: 'http://localhost:8080/api/graphql',
```
Comment (tanlethanh) [inline tools/graphql.codegen.ts:5]: please revert this url to `https://enode.peakee.co` as it is offical API endpoints, I already deployed to latest change

File: src/app/layout.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { FC, ReactNode } from 'react';
 import { Metadata, Viewport } from 'next';
 import clsx from 'clsx';
 
-import { Providers } from './Providers';
+import { Providers } from './providers';
```
Comment (tanlethanh) [inline src/app/layout.tsx:6]: is it needed?

Comment (Hoanqnqu) [reply src/app/layout.tsx:6]: Yes, my IDE shows an error

Comment (Hoanqnqu) [reply src/app/layout.tsx:6]: Yes, my IDE shows an error

**Review by tanlethanh — APPROVED** [review]

**Pull request merged**